src/Serilog.Sinks.EventLog/LoggerConfigurationEventLogExtensions.cs

```diff
@@ -57,5 +58,37 @@ public static class LoggerConfigurationEventLogExtensions
 
 			return loggerConfiguration.Sink(new EventLogSink(source, logName, formatter, machineName, manageEventSource), restrictedToMinimumLevel);
 		}
-	}
+
+        /// <summary>
+        /// Adds a sink that writes log events to the Windows event log.
+        /// </summary>
+        /// <param name="loggerConfiguration">The logger configuration.</param>
+        /// <param name="source">The source name by which the application is registered on the local computer.</param>
+        /// <param name="logName">The name of the log the source's entries are written to. Possible values include Application, System, or a custom event log.</param>
+        /// <param name="machineName">The name of the machine hosting the event log written to.  The local machine by default.</param>
+        /// <param name="manageEventSource">If false does not check/create event source.  Defaults to true i.e. allow sink to manage event source creation</param>
+        /// <param name="restrictedToMinimumLevel">The minimum log event level required in order to write an event to the sink.</param>
+        /// <param name="formatter">Formatter to control how events are rendered into the file. To control
+        /// plain text formatting, use the overload that accepts an output template instead.</param>
+        /// <returns>
+        /// Logger configuration, allowing configuration to continue.
+        /// </returns>
+        /// <exception cref="System.ArgumentNullException">loggerConfiguration</exception>
+        /// <exception cref="ArgumentNullException">A required parameter is null.</exception>
+		public static LoggerConfiguration EventLog(
+            this LoggerSinkConfiguration loggerConfiguration,
+            string source,
+            ITextFormatter formatter,
```

Comment:
For consistency with File and RollingFile, can we move this parameter up before `source` please? The IntelliSense parameter groupings (and overload selection) seemed to work a bit better if we know early on which of the two overloads is going to be used. (File example: https://github.com/serilog/serilog-sinks-file#json-formatting). May not be the case always, but since there's a precendent it'd be good to follow.